social_network/posts.py

```diff
@@ -1,33 +1,47 @@
 from datetime import datetime
 
 
-# class Post(object):
-#     def __init__(self, text, timestamp=None):
-#         pass
-#
-#     def set_user(self, user):
-#         pass
-
-
-# class TextPost(...):  # Inherit properly
-#     def __init__(self, text, timestamp=None):
-#         pass
-#
-#     def __str__(self):
-#         pass
-
-
-# class PicturePost(...):  # Inherit properly
-#     def __init__(self, text, image_url, timestamp=None):
-#         pass
-#
-#     def __str__(self):
-#         pass
-
-
-# class CheckInPost(...):  # Inherit properly
-#     def __init__(self, text, latitude, longitude, timestamp=None):
-#         pass
-#
-#     def __str__(self):
-#         pass
+class Post(object):
+    user = None
```

Comment:
since I think of a user as an object, my preference would be to put this into the Post's init, rather than treat it as a "fixed variable"